doc/classes/Node.xml

```diff
@@ -129,6 +129,7 @@
 				    child_node.get_parent().remove_child(child_node)
 				add_child(child_node)
 				[/codeblock]
+				[b]Note:[/b] If you want a child to be persisted to a [PackedScene], you must call [method set_owner] in addition to [method add_child]. This is typically relevant for [code]tool[/code] scripts (<link>https://godot.readthedocs.io/en/latest/tutorials/misc/running_code_in_the_editor.html</link>) and editor plugins (<link>https://godot.readthedocs.io/en/latest/tutorials/plugins/editor/index.html</link>). If [method add_child] is called without [method set_owner], the newly added [code]Node[/code] will not be visible in the scene tree, though it will be visible in the 2D/3D view. See <link>https://github.com/godotengine/godot/issues/34739</link> for more info.
```

Comment:
The link syntax is `[url=https://example.com]link text[/url]`.
I wouldn't link the issue (unless it was a known bug, which it is), the documentation should be sufficient in itself.

Reply:
Checks error out with "unresolved method set_owner". It should be `[method Object.set_owner]`.

Reply:
You can use `[Node]` instead of `[code]Node[/code]`, hyperlinks to the same file are properly handled by the conversion script.